ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Examples.hs

```diff
@@ -127,31 +128,30 @@ recentOffsets = [0,1,2,3,5,8,13,21,34,55,89,144,233,377,610,987,1597,2584]
 -- This is of course only useful in tests and reference implementations since
 -- this is not a realistic chain representation.
 --
-chainSyncServerExample :: forall header blk m a.
-                          ( HasHeader header
+chainSyncServerExample :: forall header m a.
+                          ( HasHeader header -- FIXME misleading language "Has"/"Is"
                           , MonadSTM m
-                          , HeaderHash header ~ HeaderHash blk
                           )
                        => a
                        -> TVar m (ChainProducerState header)
-                       -> ChainSyncServer header (Point blk) m a
+                       -> ChainSyncServer header (Point header) m a
```

Comment:
This looks like a reasonable change, but why is it in this patch? It looks unrelated.